Regression/WarpX-tests.ini

```diff
@@ -271,7 +271,7 @@ buildDir = .
 inputFile = Examples/Tests/Langmuir/inputs_3d_rt
 runtime_params = electrons.ux=0.01 electrons.xmax=0.e-6 warpx.fields_to_plot=Ex jx electrons.plot_vars=w ux Ex
 dim = 2
-addToCompileString =
+addToCompileString = USE_OPENPMD=FALSE
```

Comment:
Isn't that the default?

Reply:
not in the build matrix entries I added here.

Either way, I'll wait for #723 to land and will then refactor this PR.